README.md

```diff
@@ -44,6 +54,7 @@ However, it can be installed explicitly with the [LSST Software Build Tool](http
 
     git clone https://github.com/lsst/ap_verify_ci_hits2015/
     setup -r ap_verify_ci_hits2015
+    ap_verify.py --dataset CI-HiTS2015 --gen2 --output /path/to/desired/workspace
```

Comment:
This example implies that running `ap_verify.py` is necessary to install the dataset, which is not true (and one might, for example, want to run `ingest_dataset.py` instead). Given DM's policy on keeping documentation focused, I'd prefer that users just follow the links above and below, but could you at least separate it with a description that's it's one way to use the dataset?

Reply:
I've always been annoyed that I have to click through to find the syntax of that command, but your points are valid, and now that I think about it, this is exactly the "duplicate documentation" I was pushing you to avoid in your own PR 😆 I'll remove this but instead mention that the "name" of this dataset is `CI-HiTS2015` since that's the piece I always forget. I'll do the same in the main dataset.

Reply:
As soon as I have some time I'll try to file an RFC to remove those names. They're a pain all around.